components/contactus.php

```diff
@@ -18,7 +19,7 @@
                 </div><br>
                 <div class="col-md-6">
                     <div class="form-group">
-                        <textarea name="txtemsg" class="form-control" placeholder="message" style="width: 100%; height: 150px;"></textarea>
+                        <textarea name="txtemsg" class="form-control" placeholder="message" style="width: 100%; height: 100px;"></textarea>
```

Comment:
Try to write CSS in another class. Don't write inline CSS. Its not good for the SEO. @nethu99